rllib/utils/replay_buffers/mixin_replay_buffer.py

```diff
@@ -0,0 +1,274 @@
+                sequences sampled (B).
+            prioritized_replay_alpha: Alpha parameter for a prioritized
+                replay buffer. Use 0.0 for no prioritization.
+            prioritized_replay_beta: Beta parameter for a prioritized
+                replay buffer.
+            prioritized_replay_eps: Epsilon parameter for a prioritized
+                replay buffer.
+            replay_mode: One of "independent" or "lockstep". Determined,
+                whether in the multiagent case, sampling is done across all
+                agents/policies equally.
+            replay_sequence_length: The sequence length (T) of a single
+                sample. If > 1, we will sample B x T from this buffer.
+            replay_burn_in: The burn-in length in case
+                `replay_sequence_length` > 0. This is the number of timesteps
+                each sequence overlaps with the previous one to generate a
+                better internal state (=state after the burn-in), instead of
+                starting from 0.0 each RNN rollout.
+            replay_zero_init_states: Whether the initial states in the
+                buffer (if replay_sequence_length > 0) are alwayas 0.0 or
+                should be updated with the previous train_batch state outputs.
+            replay_ratio: Ratio of replayed samples in the returned
+                batches. E.g. a ratio of 0.0 means only return new samples
+                (no replay), a ratio of 0.5 means always return newest sample
+                plus one old one (1:1), a ratio of 0.66 means always return
+                the newest sample plus 2 old (replayed) ones (1:2), etc...
+        """
+        if not 0 < replay_ratio < 1:
+            raise ValueError("Replay ratio must be within [0, 1]")
+
+        MultiAgentReplayBuffer.__init__(self,
+                                        capacity,
+                                        storage_unit,
+                                        num_shards,
+                                        learning_starts,
+                                        replay_batch_size,
+                                        prioritized_replay_alpha,
+                                        prioritized_replay_beta,
+                                        prioritized_replay_eps,
+                                        replay_mode,
+                                        replay_sequence_length,
+                                        replay_burn_in,
+                                        replay_zero_init_states,
+                                        )
+
+        self.replay_ratio = replay_ratio
+        self.replay_proportion = None
+        if self.replay_ratio != 1.0:
+            self.replay_proportion = self.replay_ratio / (
+                1.0 - self.replay_ratio)
+
+        # Last added batch(es).
+        self.last_added_batches = collections.defaultdict(list)
+
+    @ExperimentalAPI
+    @DeveloperAPI
```

Comment:
Same here, only `@ExperimentalAPI`.
There are a couple more instances of this below.